### DISCUSSION TITLE
ProgressBar - no way for background-color?

### DISCUSSION THREAD

**Discussion by luca-sacco** [taiga-family/taiga-ui#1245] [Q&A] [answered]
Hi eveyone,
is there no way to customize the background color of the ProgressBar component?
I can't via css, it seems the only way is via variables (--tui-base-03)

**Comment by loxy** [comment, marked as answer]
These are CSS Variables: See: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

**Reply by waterplea** [reply]
That's right, you can just override this variable on the bar itself to the color you need.
